packages/quantic/force-app/main/default/lwc/quanticResult/quanticResult.js

```diff
@@ -2,16 +2,34 @@
 import defaultTemplate from './quanticResult.html';
 import { LightningElement, api } from "lwc";
 
+/** @typedef {import("coveo").Result} Result */
+/** @typedef {import("coveo").ResultTemplatesManager} ResultTemplatesManager */
+
+/**
+ * The `QuanticResult` component is used internally by the `QuanticResultList` component.
+ * @example
+ * <c-quantic-result engine-id={engineId} key={result.uniqueId} result={result} result-templates-manager={resultTemplatesManager}></c-quantic-result>
```

Comment:
keys are required in when enumerating components in the markup but they are for the dom so they aren't a component attribute. I'll remove it though as there is no context in the example that suggests this is being enumerated.

Reply:
Oh interesting, thanks
Do you think maybe it would be good to keep it in the example, and to explain in the example that a key is needed for the component to work in the DOM?

Reply:
No given this isn't specific to this component. Any html element that's rendered through iteration requires a key.